pkg/assetfetcher/config.go

```diff
@@ -0,0 +1,35 @@
+package assetfetcher
+
+import "context"
+
+// AssetFetcherConfig holds configuration to connect to Nexpose
+// and make a call to the fetch assets API
+type AssetFetcherConfig struct {
+	Host                 string
+	Username             string
+	Password             string
+	AssetFetcherPageSize int
```

Comment:
nit: It's redundant to call this `AssetFetcherPageSize` given the surrounding context of `nexpose`. I think `PageSize` would be just as effective and maybe easier to read when looking at ENV vars.

Reply:
Okay, adding the description as help information makes me feel better about this. I'll change it, Thanks